src/utils/ExtrusionJunction.cpp

```diff
@@ -20,4 +20,17 @@ ExtrusionJunction::ExtrusionJunction(const Point p, const coord_t w, const coord
       region_id(region_id)
 {}
 
+void ExtrusionJunction::Swap(ExtrusionJunction* other) {
```

Comment:
code-style bracket on newline and naming, we use lowerCamelCase for functions:
```suggestion
void ExtrusionJunction::swap(ExtrusionJunction* other)
{
```